src/website/app/demos/layout/examples/start-end/importSyntax.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+export default {
+  id: 'import-syntax',
+  title: 'Import Syntax',
+  description: `\`\`\`
+import { StartEnd } from 'mineral-ui/layout';
```

Comment:
It looks like most of the examples are using Box also. Should we include Box in the import syntax or is StartEnd useful on it's own?

Reply:
It's useful on its own. Box just provides the same example aesthetic as the other layout components.

Reply:
Ah, gotcha. Cool